package generated;
import java.util.concurrent.CompletableFuture;
import java.util.concurrent.ExecutionException;
import recaf.async.AsyncExtension;
 
public class TestAsyncNoAwait {
    private static recaf AsyncExtension<Integer> alg = new AsyncExtension<Integer>();
    
    recaf CompletableFuture<Integer> op() {
    	if (1 > 5) {
       		return 42;
     	}
     	return 41;
  	}
  	
  	public static void main(String[] args) throws InterruptedException, ExecutionException{
  		Integer answer = new TestAsyncNoAwait().op().get();
  		
  		System.out.println(answer);
  	}
}